backend/btrixcloud/colls.py

```diff
@@ -762,7 +762,10 @@ async def stream_iter():
                 flush=True,
             )
             await self.storage_ops.delete_file_object(org, thumbnail_file)
-            raise HTTPException(status_code=400, detail="upload_failed")
+            raise HTTPException(
+                status_code=400,
+                detail="Upload failed: maxiumum thumbnail size (2 MB) exceeded",
```

Comment:
would be better to have this be an id, so frontend can localize

Reply:
Yes, fair. I think we're already validating this in the frontend before submission but can write it so it's useful for API users as well.